src/runtime/virtcontainers/hypervisor.go

```diff
@@ -440,6 +440,17 @@ type HypervisorConfig struct {
 
 	// Enable annotations by name
 	EnableAnnotations []string
+
+	// PVPanicEnabled control the behaviour whether the runtime should catch "GUEST_PANICKED"
+	// and propagate it to outside the guest
```

Comment:
I'd generalise the wording a little here since this is the interface, not a QEMU specific comment. Something like:

```suggestion
	// PVPanicEnabled controls whether the runtime should detect if the guest kernel panics
	// and propagate the details to the host environment.
```